crates/amaru/src/stages/ledger.rs

```diff
@@ -115,6 +115,7 @@ impl<S: Store + Send, HS: HistoricalStores + Send> ValidateBlockStage<S, HS> {
         match rules::validate_block(&mut context, ProtocolParameters::default(), &block) {
             BlockValidation::Err(err) => return Err(err),
             BlockValidation::Invalid(err) => {
+                error!("Block invalid: {:?}", err);
```

Comment:
Not sure it fully aligns with the current span driven approach. We might want to figure out a more global way to deal with error.

Reply:
Just wanted to make sure something was getting printed for now.